packages/react-tree-grid/src/components/TreeGrid/TreeGrid.tsx

```diff
@@ -1,20 +1,18 @@
 import * as React from 'react';
-import { makeResetStyles, mergeClasses } from '@fluentui/react-components';
-
-export type TreeGridProps = JSX.IntrinsicElements['div'];
-
-const useStyles = makeResetStyles({
-  display: 'block',
-});
+import { mergeClasses } from '@fluentui/react-components';
+import { useTreeGridStyles } from './useTreeGridStyles.styles';
+import { TreeGridProps } from './TreeGrid.types';
+import { useNavigation } from '../../hooks/useNavigation';
 
 export const TreeGrid = React.forwardRef(
   (props: TreeGridProps, ref: React.ForwardedRef<HTMLDivElement>) => {
```

Comment:
This component does not follow the structure of Fluent components. What is the plan there?

Reply:
None of the components in fluent contrib does